Dockerfile.slim

```diff
@@ -0,0 +1,49 @@
+#---------------------------------------------------------------------------------------------
+# See LICENSE in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+ARG PYTHON_VERSION="3.11"
+
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /wheels
+
+RUN pip install --upgrade pip \
+    && pip install wheel
+
+COPY . .
+
+RUN pip wheel .
+
+RUN mkdir /wheels/recommended_plugins
+
+COPY ./recommended-plugins.txt ./recommended-plugins.txt
+
+RUN pip wheel .[recommended-plugins] --wheel-dir=/wheels/recommended_plugins
+
+FROM python:${PYTHON_VERSION}-slim
+
+# See http://label-schema.org for metadata schema
+# TODO: Add `build-date` and `version`
```

Comment:
Wonder if OCI is better:
https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

https://snyk.io/blog/how-and-when-to-use-docker-labels-oci-container-annotations/

Reply:
Let me know what you want to do here

Reply:
Yes, OCI is more standard now. Also http://label-schema.org is does not work on https which is sus

Reply:
It also just says it is deprecated and use OCI

Reply:
OCI added